src/store/reducers/index.js

```diff
@@ -0,0 +1,7 @@
+import { combineReducers} from 'redux';
+import lists from './list';
+import addTodoInList from './add_todo_in_list';
+export default combineReducers({
+    lists,
+    addTodoInList
+})
```

Comment:
use empty last line

Reply:
Sir iski semjh nh arhi mjy

Reply:
I meant there should be empty line at the end of each file

Reply:
otherwise github shows warning in PR

Reply:
done